server/market/orderrouter.go

```diff
@@ -208,7 +208,11 @@ func (r *OrderRouter) handleLimit(user account.AccountID, msg *msgjson.Message)
 	var commit order.Commitment
 	copy(commit[:], limit.Commit)
 
-	// Create the limit order
+	// Create the limit order.
+	force := order.StandingTiF
+	if limit.TiF == msgjson.ImmediateOrderNum {
```

Comment:
`limit.TiF` already checked on https://github.com/decred/dcrdex/pull/304/files#diff-b3757e15aa4e22aa99eab9a723d0713dR200 to be one of `msgjson.StandingOrderNum` or `msgjson.ImmediateOrderNum`